ruby_on_rails/rails_basics/deployment.md

```diff
@@ -207,24 +221,23 @@ This will be where the Git skills you've been learning will start to really pay
 <div class="lesson-content__panel" markdown="1">
 
 1. Deploy your [Blog App project](https://www.theodinproject.com/lessons/ruby-on-rails-blog-app) to one of the hosting providers we've mentioned. If you need help deciding which one to use, we recommend Fly.io. The important thing to take away from this first deployment is getting experience deploying. Don't worry if you don't understand everything that's happening. That will come with time.
-    * Use one of the linked deploy guides for your PaaS provider to help you through the process.
-    * If you're having trouble deploying, check out the [Debugging and Troubleshooting Deployments](#debugging-and-troubleshooting-deployments) section for some tips.
+_ Use one of the linked deploy guides for your PaaS provider to help you through the process.
+_ If you're having trouble deploying, check out the [Debugging and Troubleshooting Deployments](#debugging-and-troubleshooting-deployments) section for some tips.
```

Comment:
```suggestion
   - Use one of the linked deploy guides for your PaaS provider to help you through the process.
   - If you're having trouble deploying, check out the [Debugging and Troubleshooting Deployments](#debugging-and-troubleshooting-deployments) section for some tips.
```

These need dashes instead of underscore, and a [3 spacing indent for nesting inside a numbered list](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#nested-lists)